packages/multichain-account-service/src/MultichainAccountService.ts

```diff
@@ -293,6 +302,55 @@ export class MultichainAccountService {
     return Array.from(this.#wallets.values());
   }
 
+  /**
+   * Creates a new multichain account wallet with the given mnemonic.
+   *
+   * NOTE: This method should only be called in client code where a mutex lock is acquired.
+   * `discoverAndCreateAccounts` should be called after this method to discover and create accounts.
+   *
+   * @param options - Options.
+   * @param options.mnemonic - The mnemonic to use to create the new wallet.
+   * @throws If the mnemonic has already been imported.
+   * @returns The a tuple of the new multichain account wallet and the entropy source id.
+   */
+  async createMultichainAccountWallet({
+    mnemonic,
+  }: {
+    mnemonic: string;
+  }): Promise<MultichainAccountWallet<Bip44Account<KeyringAccount>>> {
+    const existingKeyrings = this.#messenger.call(
+      'KeyringController:getKeyringsByType',
+      KeyringTypes.hd,
+    ) as (EthKeyring & { mnemonic: Uint8Array })[];
```

Comment:
Type-wise, this field could be `null` or `undefined`, [see here](https://github.com/MetaMask/accounts/blob/main/packages/keyring-eth-hd/src/hd-keyring.ts#L107).

In practice, I assume it's always defined, but maybe we want to align the type here too? We could also have `eth-hd-keyring` as a dev dep and use the `HdKeyring` type here too.

Which would force to handle the "null case".

WDYT?